crates/wash-lib/src/parser/mod.rs

```diff
@@ -109,14 +109,18 @@ struct RawActorConfig {
 impl TryFrom<RawActorConfig> for ActorConfig {
     type Error = anyhow::Error;
 
-    fn try_from(raw_config: RawActorConfig) -> Result<Self> {
+    fn try_from(raw_config: RawActorConfig) -> Result<Self, Self::Error> {
+        let key_dir = if let Some(key_directory) = raw_config.key_directory {
+            key_directory
+        } else {
+            let home_dir = dirs::home_dir()
+                .ok_or_else(|| anyhow::anyhow!("Unable to determine the user's home directory"))?;
+            home_dir.join(".wash/keys")
+        };
         Ok(Self {
             claims: raw_config.claims.unwrap_or_default(),
             push_insecure: raw_config.push_insecure.unwrap_or(false),
-            // TODO(#1624): Default to ~/.wash/keys
-            key_directory: raw_config
-                .key_directory
-                .unwrap_or_else(|| PathBuf::from("./keys")),
+            key_directory: key_dir,
```

Comment:
nit: if you call this variable `key_directory` the last line here is simpler with the syntactic sugar.
```rust
let key_directory = ...
...
        Ok(Self {
            claims: raw_config.claims.unwrap_or_default(),
            push_insecure: raw_config.push_insecure.unwrap_or(false),
            key_directory,
```

Reply:
great catch. updated to a more cleaner code :)